packages/sdks/typescript/src/index.ts

```diff
@@ -1,6 +1,9 @@
 import type { ContentType, Message, MessageRole } from '@latitude-data/compiler'
-import { DocumentLog, type ChainEventDto } from '@latitude-data/core/browser'
-import { EvaluationResultDto } from '@latitude-data/core/repositories'
```

Comment:
Was this a problem in some way @geclos? I though all we get from core in terms of types have to be in browser. Maybe this was before. I don't remember.

Reply:
if it's only types it's fine, issue is if you start importing enums or other code that needs to be evaluated by rollup